src/renderer/components/uielements/assets/assetCard/AssetCard.style.ts

```diff
@@ -110,3 +99,11 @@ export const InputBigNumber = styled(BaseInputBigNumber).attrs({
 export const SliderWrapper = styled.div`
   padding: 0 5px 20px 10px;
 `
+
+export const AssetInfo = styled(AssetDataUI)`
+  padding-top: 10px;
+  & .ticker {
+    font-size: 16px;
+  }
+  border-bottom: 1px solid ${palette('gray', 0)};
```

Comment:
AssetData is a component with stuff we don't need (icon, price etc.). Instead of using AssetData, we should create a simple (light-weight) component to display ticker + chain only. 

^ See #1136

It will be another PR.